python/lib/core/dmod/core/context/server.py

```diff
@@ -0,0 +1,128 @@
+"""
+Defines a custom implementation for a multiprocessing context Server
+
+Overcomes issues present in the current version of Python (v3.8) and provides extra functionality, such as
+advanced handling for properties
+"""
+from __future__ import annotations
+
+import typing
+import threading
+import sys
+
+from multiprocessing import managers
+from multiprocessing import util
+
+from traceback import format_exc
+
+from ..decorators import version_range
+
+from .base import is_property
+
+
+@version_range(
+    maximum_version="3.12.99",
+    message="Python Versions 3.12 and below have an issue where raw values cannot be returned by a Server if the "
+            "named value wasn't a function. Check if this is still an issue now that a more recent version is used."
+)
+class DMODObjectServer(managers.Server):
```

Comment:
I added a wild amount of comments and disclaimers explaining what each part is, what it does, where it came from, and what to reference.

This class was not added lightly. Proceeding without this would require the implementation of non-pythonic code elsewhere and a necessary rewrite of important structures.

Property functionality is a bonus. What is _really_ important is `value_or_function = getattr(served_object, member_name)` followed by `if isinstance(value_or_function, typing.Callable):`.